libcudacxx/docs/extended_api/memory_model.md

```diff
@@ -53,10 +53,10 @@ the scope is `thread_scope_system` and:
 
 - it affects an object in [unified memory] and [`concurrentManagedAccess`] is `1`, **or**
 - it affects an object in CPU memory and [`hostNativeAtomicSupported`] is `1`, **or**
-- it is a load or store that affects a naturally-aligned object of sizes `1`, `2`, `4`, or `8` bytes on [mapped memory], **or**
+- it is a load or store that affects a naturally-aligned object of sizes `1`, `2`, `4`, `8`, or `16` bytes on [mapped memory], **or**
```

Comment:
We should try and make sure that it is obvious that `16` is only 12.4 onwards

Reply:
There are no 16-byte wide system scope atomic loads or stores before that version, i.e., this holds retroactively for all CUDA versions, and that includes from the version in which this operations were made available, and onwards.